conans/client/downloaders/local_file_downloader.py

```diff
@@ -0,0 +1,29 @@
+import os
+from urllib.parse import urlparse
+from urllib.request import url2pathname
+from shutil import copyfile
+
+from conans.client.tools.files import check_md5, check_sha1, check_sha256
+from conans.errors import ConanException
+
+
+class LocalFileDownloader(object):
+
+    def __init__(self, output):
+        self._output = output
+
+    def download(self, url, file_path, md5=None, sha1=None, sha256=None, **kwargs):
+
+        file_origin = self._path_from_file_uri(url)
+        copyfile(file_origin, file_path)
```

Comment:
Note here: `shutil.copyfile()` will overwrite the file if it already exists - I've not added any logic to handle that case. From what I can see, in the newer implementation we are defaulting to `overwrite=True` always, and the `overwrite` argument is no longer exposed.
See here: https://github.com/conan-io/conan/blob/develop/conan/tools/files/files.py#L151

But I can see the file downloader does interpret this flag:
https://github.com/conan-io/conan/blob/develop/conans/client/downloaders/file_downloader.py#L46-L53

If there are plans to make overwrites configurable (via `conf` like with the other options), then it's worth implementing now